org.eclipse.dawnsci.hdf5.editor/src/org/eclipse/dawnsci/hdf5/editor/ServiceHolder.java

```diff
@@ -28,15 +28,15 @@ public static ILoaderService getLoaderService() {
 		return loaderService;
 	}
 
-	public static void setLoaderService(ILoaderService loaderService) {
+	public void setLoaderService(ILoaderService loaderService) {
 		ServiceHolder.loaderService = loaderService;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)